integration-cli/docker_cli_build_test.go

```diff
@@ -1986,8 +1986,7 @@ func TestBuildCancelationKillsSleep(t *testing.T) {
 	eventStart := make(chan struct{})
 	eventDie := make(chan struct{})
 
-	// Start one second ago, to avoid rounding problems
-	startEpoch := time.Now().Add(-1 * time.Second)
+	startEpoch := daemonTime(t).Unix()
```

Comment:
@duglin I don't know what deamonTime has to do with race stuff, I'm almost done fixing the race in this test btw